docker/database/initial_db_setup.sql

```diff
@@ -1 +1,2 @@
-CREATE DATABASE IF NOT EXISTS hs_hub;
\ No newline at end of file
+CREATE DATABASE IF NOT EXISTS hs_hub;
+CREATE DATABASE IF NOT EXISTS hs_discord;
```

Comment:
Is this here for future changes you are going to make? I can't see any references to the `hs_discord` database anywhere

Reply:
It's used by the `hs_discord_api_repo`, I'll move it there instead :+1: